src/Components/ClusterHeader/ClusterHeader.js

```diff
@@ -20,9 +23,31 @@ export const ClusterHeader = ({ clusterId, clusterData }) => {
     data: cluster,
   } = clusterData;
 
+  const dropDownToggle = (isOpen) => {
+    setIsOpen(isOpen);
+  };
+
+  const dropDownSelect = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const redirectOCM = (clusterId) => {
+    location.replace(
```

Comment:
This creates one unwanted behavior: once you click on the button and are redirected to the new page, if you click the browser's "Back", you won't get to the single cluster page in Advisor from where you were navigated to OCM, but to the page before. This is caused by the usage of `replace` which literally replaces your current URL with a new one instead of adding a new one on top of it.
https://developer.mozilla.org/en-US/docs/Web/API/Location/replace

Reply:
Feel free to use direct assignment to `window.location`:
```
const redirectOCM = (clusterId) => {
    window.location =
      window.location.origin +
      (window.location.pathname.includes('beta') ? `/beta` : '') +
      `/openshift/details/${clusterId}`;
  };
```